src/seedu/addressbook/data/person/ReadOnlyPerson.java

```diff
@@ -66,16 +67,17 @@ default String getAsTextShowAll() {
      * Formats a person as text, showing only non-private contact details.
      */
     default String getAsTextHidePrivate() {
+        final Person p = getPerson();
         final StringBuilder builder = new StringBuilder();
         builder.append(getName());
         if (!getPhone().isPrivate()) {
-            builder.append(" Phone: ").append(getPhone());
+            builder.append(p.getPrintableString(p.getPhone()));
         }
         if (!getEmail().isPrivate()) {
-            builder.append(" Email: ").append(getEmail());
+            builder.append(p.getPrintableString(p.getEmail()));
         }
         if (!getAddress().isPrivate()) {
-            builder.append(" Address: ").append(getAddress());
+            builder.append(p.getPrintableString(p.getAddress()));
```

Comment:
Great to see that you invoke the new method getPrintableString() . 
Please note that going forward you need to update the tests as well when you make any behaviorial changes to your code.